docs/user-wallets/overview.md

```diff
@@ -0,0 +1,80 @@
+import StateOfTheNetwork from "@site/src/pages/state-of-the-network.md";
+import FeedbackComponent from "@site/src/pages/feedback.md";
+
+# Introduction to User Wallets
+
+<StateOfTheNetwork/>
+
+:::info
+Need some `testLPX` test tokens to mint a PKP on Chronicle? Use the faucet: https://faucet.litprotocol.com/
+:::
+
+## Introduction
+
+The Lit Protocol offers two novel ways of managing private keys with [Programmable Key Pairs (PKPs)](./pkps/overview.md) and [Wrapped Keys](./wrapped-keys/overview.md). Using either of these two methods, you can provide your users with a seamless onboarding experience to your decentralized application, all while offering them non-custodial wallets with strict access control.
```

Comment:
> Using either of these two methods, you can provide your users with a seamless onboarding experience to your decentralized application, all while offering them non-custodial wallets with strict access control.

consider "either of these methods can be used to provide users of your application with a seamless, secure, and non-custodial onboarding and asset management experience"